src/libs/ajax/DatasetBuilder.ts

```diff
@@ -69,23 +95,88 @@ export interface GetConceptsResponse {
 type DatasetRequest = {
   cohorts: Cohort[];
   conceptSets: ConceptSet[];
-  valuesSets: { domain: string; values: DatasetBuilderValue[] }[];
+  valueSets: { domain: string; values: DatasetBuilderValue[] }[];
+};
+
+export type DatasetRequestAPI = {
+  cohorts: CohortAPI[];
+  conceptSets: ConceptSet[];
+  valueSets: { id: number; name: string; values: string[] }[];
 };
 
-type DatasetAccessRequest = {
+export type DatasetAccessRequest = {
   name: string;
   researchPurposeStatement: string;
   datasetRequest: DatasetRequest;
 };
 
+export type DatasetAccessRequestAPI = {
+  name: string;
+  researchPurposeStatement: string;
+  datasetRequest: DatasetRequestAPI;
+};
+
+export const convertValueSets = (valueSets: { domain: string; values: DatasetBuilderValue[] }[]) => {
+  return _.map(
+    (valueSet) => ({ id: 0, name: valueSet.domain, values: _.map(({ name }) => name, valueSets.values) }),
```

Comment:
Is `valueSets.values` correct? I would expect this to be `valueSet.values`.
```suggestion
    (valueSet) => ({ id: 0, name: valueSet.domain, values: _.map(({ name }) => name, valueSet.values) }),
```

Reply:
Also I believe `_.map('name', valueSet.values)` is the same as `.map(({ name }) => name, valueSet.values)`